src/Microsoft.VisualStudio.Editors/PropPages/AssemblyInfoPropPage.vb

```diff
@@ -172,6 +172,27 @@ Namespace Microsoft.VisualStudio.Editors.PropertyPages
             Common.SetComboBoxDropdownWidth(NeutralLanguageComboBox)
         End Sub
 
+        Private _neutralLanguageWasDroppedDown As Boolean = False
+
+        ''' <summary>
+        ''' For checking if the NeutralLanguageComboBox was DroppedDown when the escape key was pressed.
+        ''' With a combination of AutoCompleteMode.SuggestAppend and AutoCompleteSource.ListItems the 
+        ''' NeutralLanguageComboBox and the Assembly Info window will both handle a Escape key press and both close.
+        ''' We only want the NeutralLanguageComboBox to close in that case.
+        ''' </summary>
+        Private Sub NeutralLanguageComboBox_PreviewKeyDown(sender As Object, e As PreviewKeyDownEventArgs) Handles NeutralLanguageComboBox.PreviewKeyDown
+            If (e.KeyData And Keys.KeyCode) = Keys.Escape And NeutralLanguageComboBox.DroppedDown Then
```

Comment:
I'm a bit surprised this is necessary, but I can repro it in a standalone app.  Have we filed a bug on Winforms to see if this is by design?

I would have thought that if the combo box had focus it would handle the escape before it got to the dialog.

Reply:
I was very surprised as well. I will be filing a bug for this.